src/.vitepress/theme/index.ts

```diff
@@ -2,7 +2,7 @@ import { EnhanceAppContext } from 'vitepress'
 import DefaultTheme from 'vitepress/theme'
 import { h } from 'vue'
 import './index.css'
-import Publications from '../components/Publications.vue'
+import {Publications} from '../components/Publications'
```

Comment:
Remove braces?

Reply:
Publications is a named export so we need do this